Graphs/FordFulkerson.js

```diff
@@ -0,0 +1,52 @@
+// https://en.wikipedia.org/wiki/Ford%E2%80%93Fulkerson_algorithm
+function fordFulkerson(capacity, source, sink) {
+  const V = capacity.length;
+  const residualCapacity = capacity.map((arr) => arr.slice());
+  const parent = Array(V).fill(-1);
+  let maxFlow = 0;
+  
+  function bfs(source, sink) {
```

Comment:
Since we're using BFS to find augmenting paths, this isn't just any Ford-Fulkerson algorithm, it's the Edmonds-Karp variant to be specific (which has a runtime bound independent from the specific capacities).